src/NServiceBus.RavenDB/Shutdown/ShutdownHook.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.RavenDB.Shutdown
+{
+    /// <summary>
+    /// This class is to be ported to the Endpoint level in NServiceBus core.
```

Comment:
I'm going to remove this comment since this is not the direction Core is going.

Correct @colin-higgins ?